src/routes/savoringYourChild/SavoringYourChild.js

```diff
@@ -32,7 +32,7 @@ const SavoringYourChild = ({ user, typeformUserId }: Props) => (
       <Typeform
         data-url={`https://adamgoldman.typeform.com/to/VHYYNS?typeformUserId=${typeformUserId}`}
         style={{ width: '100%', height: 500 }}
-        onSubmit={() => console.log('submit!')}
+        onSubmit={() => { window.location.href = '/savoring-your-child/peaceful-ending' }}
```

Comment:
look [here](https://github.com/kriasoft/react-starter-kit/blob/master/docs/recipes/how-to-implement-routing.md) how they suggest doing it without a page refresh (we never refresh the page)

look for this: `history.push()`